src/test/util_test.ts

```diff
@@ -4,13 +4,15 @@ import {expect} from 'chai';
 import * as ESTree from 'estree';
 import {AST, Rule} from 'eslint';
 
-const parseExpr = (expr: string): ESTree.Node => {
+const parseExpr = (expr: string): ESTree.Class => {
```

Comment:
i'd leave this one myself, it works right now but the idea was that it could be used in any future tests to quickly parse some source, not specific to classes. just so happened we haven't used it for that yet, but that was the intent